src/riscv.c

```diff
@@ -148,3 +167,135 @@ void rv_reset(riscv_t *rv, riscv_word_t pc)
 
     rv->halt = false;
 }
+
+#if defined(__APPLE__)
```

Comment:
Remove the inline functions `isinff` and `isnanf`. The function definitions should appear in `src/riscv_private.h`.